{"name": "test_invalid_login", "status": "failed", "statusDetails": {"message": "AssertionError: Expected error message not shown\nassert '' != ''", "trace": "driver = <appium.webdriver.webdriver.WebDriver (session=\"2a11d687959d5b6af9c27ac68c33c396d5f0e138\")>\n\n    def test_invalid_login(driver):\n        lp = LoginPage(driver)\n        # known invalid user in sample app\n        lp.login(\"locked_out_user\", \"secret_sauce\")\n        err = lp.get_error()\n>       assert err != \"\", \"Expected error message not shown\"\nE       AssertionError: Expected error message not shown\nE       assert '' != ''\n\ntests\\test_login.py:16: AssertionError"}, "attachments": [{"name": "screenshot-android", "source": "5f548d28-f335-4018-943a-b3c50bdacfc2-attachment.png", "type": "image/png"}], "start": 1757862570797, "stop": 1757862580750, "uuid": "97f9dd64-2e33-46f8-aebd-639dca1d1e07", "historyId": "6485cd1aa206d185bd707f422a9a9238", "testCaseId": "6485cd1aa206d185bd707f422a9a9238", "fullName": "tests.test_login#test_invalid_login", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "host", "value": "DB-L-174"}, {"name": "thread", "value": "8152-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}], "titlePath": ["tests", "test_login.py"]}